### DISCUSSION TITLE
Your Fork DailyDesktopWallpaperPlus on GitHub

### DISCUSSION THREAD

**Discussion by ghost** [community/community#35611] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi codecopy

I noticed that you forked my repository DailyDesktopWallpaperPlus_macOS.
I deleted my old user account pgc062020.
my real name is still there.

My request to you:
Please delete your Fork of my deleted Repository DailyDesktopWallpaperPlus_macOS.

Thank you in advance.

Best regards

**Comment by queenofcorgis** [comment, marked as answer]
Hi there! 

Thanks for reaching out. I just wanted to note it sounds like you want to contact a specific user - you may want to open an Issue on their repository instead as this is a public forum for all GitHub users 😄 .
